examples/projects/shared-volume.yml

```diff
@@ -0,0 +1,44 @@
+# An example to test a Python based project triggered by GitHub Pull Requests
+# This can be tested under the simulator with:
+#   ./simulator.py create \
+#       -d ./examples/projects/persistent-volume.yml \
+#       -t github \
+#       -r unit-test \
+#       -w /tmp/persistent-volume-simulator \
+#       -p GH_OWNER=linaro-technologies \
+#       -p GH_REPO=jobserv \
+#       -p GH_PRNUM=8 \
+#       -s githubtok=<your github personal access token>
+timeout: 5
+triggers:
+  - name: github
+    type: github_pr
+    runs:
+      - name: unit-test
+        container: python:3.5-alpine
+        host-tag: amd64
+        script: unit-test
+        shared-volumes:
+          # The runner will mount a read-only shared volume for this run
+          venv_cache-foo.12: /var/cache-venv
+      - name: flake8
+        container: python:3.5-alpine
+        script: unit-test
+        host-tag: amd64
+        script: flake8
+    email:
+      # this tells the JobServ to send an email when a build completes (ie
+      # all the runs)
+      users: 'andy.doan@linaro.org, tyler.baker@linaro.org'
+      only_failures: True
+
+scripts:
+  flake8: |
+    #!/bin/sh -ex
+    pip3 install flake8
+    flake8 --exclude=migrations/ ./
+
+  unit-test: |
+    #!/bin/sh -ex
+    echo "This is persistent" >> /var/cache-venv/tmp.log
```

Comment:
"This is shared" instead of  "This is persistent", right?

Reply:
If runner mount volume in RO-mode, this echo will fail. Am I right?

Reply:
force-push fixed